From 6fcb9ec524e6855e043ba2e96c31af39c6d51abf Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:46 -0400
Subject: [redhat] kabi: Adding symbol blk_stop_queue (block/blk-core.c)

Message-id: <0579fef1b0e1efff4683493c7a7e5ed7542e4b5f.1410510773.git.poros@redhat.com>
Patchwork-id: 95918
O-Subject: [PATCH RHEL7.1 046/233] kabi: Adding symbol blk_stop_queue (block/blk-core.c)
Bugzilla: 1062361
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Needed by PowerPath to stop processing of a queue when PowerPath is stopped on a host.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/blk_stop_queue
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/blk_stop_queue
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/blk_stop_queue

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/blk_stop_queue b/redhat/kabi/kabi-module/kabi_ppc64/blk_stop_queue
new file mode 100644
index 0000000..1b2365e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/blk_stop_queue
@@ -0,0 +1,2 @@
+#1-
+0x08795d9d blk_stop_queue vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/blk_stop_queue b/redhat/kabi/kabi-module/kabi_s390x/blk_stop_queue
new file mode 100644
index 0000000..8791de9
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/blk_stop_queue
@@ -0,0 +1,2 @@
+#1-
+0xf35aeff5 blk_stop_queue vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/blk_stop_queue b/redhat/kabi/kabi-module/kabi_x86_64/blk_stop_queue
new file mode 100644
index 0000000..9fbd027
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/blk_stop_queue
@@ -0,0 +1,2 @@
+#1-
+0x24e6fc92 blk_stop_queue vmlinux EXPORT_SYMBOL
-- 
1.7.1